keps/sig-auth/3766-referential-authorization/README.md

```diff
@@ -82,59 +94,88 @@ been [used by
 sig-storage](https://kubernetes.io/blog/2023/01/02/cross-namespace-data-sources-alpha/)
 to enable cross-namespace data sources.
 
-This KEP proposes moving ReferenceGrant from its current
-`gateway.networking.k8s.io` API group into the `authorization.k8s.io` API
-group.
+This KEP proposes building that overall idea of referential authorization
+directly into Kubernetes. This will build on the original idea of ReferenceGrant
+but add additional capabilities along the way.
 
 ## Motivation
 
-Any project that wants to enable cross-namespace references currently has to choose
-between introducing a dependency on Gateway API's ReferenceGrant or creating a
-new API that would be partially redundant (leading to confusion for users).
-	
-Recent interest between SIGs has made it clear that ReferenceGrant is wanted for use
-cases other than Gateway API. We would like to move ReferenceGrant to a neutral home
-(ideally, under SIG Auth) in order to make it the canonical API for managing references
-across namespaces.
+Any project that wants to enable cross-namespace references currently has to
+choose between introducing a dependency on Gateway API's ReferenceGrant or
+creating a new API that would be partially redundant (leading to confusion for
+users).
+
+Recent interest between SIGs has made it clear that ReferenceGrant is wanted for
```

Comment:
Could you mention which SIGs have been interested in this approach? Storage IIRC, maybe network?

Reply:
Flux also intends to consume this API.

https://github.com/fluxcd/flux2/pull/2092#issuecomment-1479788234

Reply:
Added, thanks!